examples/n-ary-app.kl

```diff
@@ -16,45 +17,53 @@
               [[cons arg0 more-args]
                [pure
                 [cons-list-syntax
-                  [quote my-app]
+                  'my-app
                   [cons-list-syntax
-                    [list-syntax [[quote #%app] fun arg0] stx]
+                    [list-syntax ['#%app fun arg0] stx]
                     more-args
                     stx]
                   stx]]])])])]]
+   -- (my-lam (x y z) ...)
```

Comment:
I would write this
```
(my-lam (x y z) body) => (lambda (x) (lambda (y) (lambda (z) body))) 
```
because I worry that the `...` would mean to readers that the argument list could be repeated 0 or more times.

Reply:
fixed